code/modules/mob/living/simple_animal/friendly/axolotl.dm

```diff
@@ -0,0 +1,32 @@
+/mob/living/simple_animal/axolotl
+	name = "axolotl"
+	desc = "Quite the colorful amphibian!"
+	icon_state = "axolotl"
+	icon_living = "axolotl"
+	icon_dead = "axolotl_dead"
+	maxHealth = 10
+	health = 10
+	attack_verb_continuous = "nibbles"
+	attack_verb_simple = "nibble"
+	melee_damage_lower = 0
+	melee_damage_upper = 0 //their teeth are just for gripping food, not used for self defense nor even chewing
```

Comment:
This is default on all simple animals, but there's a ``harm_intent_damage`` if ever it's needed.